app/test/socobo-inventory/socobo-inventory-create.html

```diff
@@ -66,7 +67,9 @@
       });
 
       test('create button is hidden when in edit mode', function(done) {
-        dialog.show({});
+        var item = {measurement: {value: 'pcs'}, desc: 'test item', info: 'test info'};
```

Comment:
@hetec -> "Because of that the measurements aren't displayed in the edit dialog" this should be fixed in my commits from yesterday.